debug.log

```diff
@@ -0,0 +1,2 @@
+[0918/142729.458:ERROR:registration_protocol_win.cc(103)] CreateFile: The system cannot find the file specified. (0x2)
```

Comment:
is this an accidental commit?

Reply:
VS Code keeps spitting these out for me... oops...